pandas/tests/indexes/test_base.py

```diff
@@ -400,7 +400,7 @@ def test_astype(self):
         casted = self.intIndex.astype('i8')
         self.assertEqual(casted.name, 'foobar')
 
-    def test_equals(self):
+    def test_equals_object(self):
         # same
```

Comment:
this tests doesn't propogate to the other tests classes. So maybe need a more generic one?